mods/d2k/tilesets/arrakis.yaml

```diff
@@ -6049,3 +6049,236 @@ Templates:
 		Tiles:
 			0: Rough
 			1: Rough
+	Template@1085:
+		Id: 1085
+		Images: BLOXXMAS.R8
+		Frames: 345, 346
+		Size: 2,1
+		Category: Bridge
```

Comment:
You need to add this category to `EditorTemplateOrder` to make them available in the editor (I suggest placing it before `Unidentified`).